interface/ceed-elemrestriction.c

```diff
@@ -638,11 +680,73 @@ int CeedElemRestrictionCreateStrided(Ceed ceed, CeedInt num_elem, CeedInt elem_s
   return CEED_ERROR_SUCCESS;
 }
 
+/**
+  @brief Create a points CeedElemRestriction, for restricting for restricting from a all local points to the current element in which they are
+ located.
+
+  The offsets array is arranged as
+
+  element_0_start_index
+  element_1_start_index
+  ...
+  element_n_start_index
+  element_n_stop_index
+  element_0_point_0
+  element_0_point_1
+  ...
```

Comment:
Of note here - I'm using a slightly different convention than in the PETSc example - the CeedElemRestriction knows the num_elem, so I removed that from the offsets array here.